tasks/InsightAppSec/task.ts

```diff
@@ -17,6 +17,13 @@ async function run() {
         var hasScanGating = tl.getBoolInput("hasScanGating");
         var generateFindingsReport = tl.getBoolInput("generateFindingsReport");
 
+        var debugModeStr = tl.getVariable("system.debug");
```

Comment:
Is this available in both build and release pipelines?

Reply:
Yes it is, need to add system.debug=true pipeline variable in the config